Automattic/jetpack
# Super Cache: Remove tracking

### Description
Updates the old links in the yellow boxes so they do not go through the jetpack redirect manager which counts redirects and therefore could be deemed as tracking.

#### Changes proposed in this Pull Request:

* Updates link behaviour

#### Other information:

- [N/A ] Have you written new tests for your changes, if applicable? 
- [ N/A] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion
N/A quick fix

#### Does this pull request change what data or activity we track or use?
Removes redirect manager links which could be deemed as tracking

#### Testing instructions:
* Check the links in the yellow box go to the right place
* Check the the links at the bottom of the easy tab go to the right place
